spec/bowling_game_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative '../lib/bowling_game'
+
+describe 'the behaviour of the game' do 
+    it "creates a bowling game" do
+    game = BowlingGame.new
+  end
+
+    it "is able to roll a gutter game" do
+        game = BowlingGame.new
+        20.times{game.roll 0}
+        expect(game.score).to eq 0
+  end
+    
+    it 'is able to roll all 1s' do
+        game = BowlingGame.new
+        20.times{game.roll 1}
+        expect(game.score).to eq 20
+    end
+
+    it "is able to roll a spare" do
+        game = BowlingGame.new
+        @game.roll 3
+        @game.roll 7
+        @game.roll 3
+        17.times{@game.roll 0}
+        expect(@game.score).to eq 16
+    end
+
+    it "is able to roll a strike" do
+         game = BowlingGame.new
+         @game.roll 10
+         @game.roll 4
+         @game.roll 3
+         16.times(@game.roll 0)
+         expect(@game.score).to eq 24
```

Comment:
This looks really good, nice succinct way of writing the tests. To avoid repetition of 'game = Bowling.new', you could have done:

'before do 
  @game = Bowling.new
end'

at the top of your code, that way you could have just called '@game' in each test instead of calling 'game = Bowling.new' each time :D